docs/administration/security/hardening-octopus.md

```diff
@@ -253,6 +253,22 @@ The TCP ports listed below are defaults, and can be changed if required - refer
 |WinRM-HTTP|`TCP 5985`|Octopus Server|Anywhere|DENY|Prevents attackers from using the Octopus Server as a beachhead into your network via unsecured WinRM.|
 |All outbound|`ALL`|Octopus Server|Anywhere|???|Depends on how fine-grained control you want over what your Octopus Server can do. It also depends on where your workers are, and where you are deploying to. Allowing all outbound traffic is a good place to start, then perform network analysis to decide on your next step.|
 
+## Harden your containers
+
+If you run an [Octopus Deploy container](/docs/installation/octopus-in-container/index.md), in addition to your usual security measure for running apps out of containers, take the following steps to secure it: 
+
+- Store your containers on a separate partition.
```

Comment:
Are you talking about the volumes? the images?
_I'm not even sure how to do this..._ (not that I'm anywhere near an expert)

Reply:
I reckon that advice (assume it comes from here: https://www.secjuice.com/how-to-harden-docker-containers/) has got to be referring to the docker data directory that lives in `/var/lib/docker`. By moving this to a different partition, it makes it harder for a compromised container to then reach into other places in `/var` (I think)

Reply:
![image](https://user-images.githubusercontent.com/579657/79316730-e59c8d00-7f43-11ea-8852-66d3e65413cd.png)

Reply:
How to move that folder:
https://www.guguweb.com/2019/02/07/how-to-move-docker-data-directory-to-another-location-on-ubuntu/